browser/brave_wallet/brave_wallet_prefs_unittest.cc

```diff
@@ -274,4 +276,79 @@ TEST_F(BraveWalletPrefsUnitTest, MigrateShowTestNetworksToggle) {
   EXPECT_FALSE(GetPrefs()->HasPrefPath(kShowWalletTestNetworksDeprecated));
 }
 
+TEST_F(BraveWalletPrefsUnitTest, MigrateShowChainIdNetworkInfo) {
+  GetPrefs()->SetBoolean(kBraveWalletUserAssetEthContractAddressMigrated, true);
+  GetPrefs()->SetBoolean(kBraveWalletUserAssetsAddPreloadingNetworksMigrated,
+                         true);
+  GetPrefs()->SetBoolean(kBraveWalletUserAssetsAddIsNFTMigrated, true);
+  GetPrefs()->SetBoolean(kBraveWalletEthereumTransactionsCoinTypeMigrated,
+                         true);
+  GetPrefs()->SetBoolean(kBraveWalletDeprecateEthereumTestNetworksMigrated,
+                         true);
```

Comment:
I leaved only one, to disable previously implemented migration.  It simplt takes all existing transactions and move them under ethereum node, ex.:
![image](https://user-images.githubusercontent.com/118171981/221684935-a87eee97-ac0b-4a0b-847c-a794508b017f.png)
and in result, the latest migration "MigrateAddChainIdToTransactionInfo" fails because JSON is wrong